tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py

```diff
@@ -449,3 +449,110 @@ def process(self, raw, identifiers, frame_meta):
                                               detections['x_mins'], detections['y_mins'], detections['x_maxs'],
                                               detections['y_maxs']))
         return result
+
+
+class SSDTFAdapter(Adapter):
+    __provider__ = 'ssd_tf'
```

Comment:
what the main difference between this adapter and ssd_multilabel?
I see that they look very similar, except for filtering background label.
Is it possible to combine these adapters into one?

Reply:
Done